comiccrawler/core/__init__.py

```diff
@@ -672,3 +680,10 @@ def analyze_info(mission, mod):
 	remove_duplicate_episode(mission)
 
 	print("Analyzing success!")
+
+def format_number(title, format):
+	"""第3卷 --> 第003卷"""
+	def replacer(match):
+		number = match.group()
+		return format.format(int(number))
+	return re.sub("\d+", replacer, title)
```

Comment:
Adding new API to format title is the safest, but let's consider if there is an universal solution first.

With above solution,
```
泰安路47号 第006回
原作版108(7)
第28-29話
```
becomes
```
泰安路47号 第006回
原作版108(007)
第028-029話
```
which looks good to me.